docs/user-guide/image_alignment.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Alignment\n",
+    "\n",
+    "In case of displacement of the detector component, i.e. scintilator,\n",
+    "the open beam (background) image can not be used\n",
+    "for normalization of the sample (signal) image without alignment.\n",
+    "\n",
+    "Here is a simple image position shifting solution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "ob_image = Path('background.tiff')\n",
+    "misaligned_image = Path('misaligned_sample.tiff')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load tiff image\n",
+    "from PIL import Image\n",
+    "\n",
+    "ob = Image.open(ob_image)\n",
+    "misaligned = Image.open(misaligned_image)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "ob_normalized = (ob / np.max(ob) * 255).astype(np.uint8)\n",
+    "misaligned_normalized = (misaligned / np.max(misaligned) * 255).astype(np.uint8)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def shift_pixels(image: np.ndarray, shift_x: int = 0, shift_y: int = 0) -> np.ndarray:\n",
```

Comment:
I don't want to sound too harsh, but I think this is the wrong direction (but I know it's not always obvious when trying to figure these things out alone at a beamline).

All data, whether it is events in a nexus file or a tiff image, should be loaded as a `DataArray`, with a `position` coordinate for the pixel positions (and other coordinates, eg `tof`?), and counts as the `data` part of the data array.

Then, shifting the positions simply becomes a vector addition.
```Py
shift = sc.vector([-0.1, 0.25, 0.], unit='cm')
da.coords['position'] += shift
```

So I see limited value in this notebook?

Reply:
(And you don't sound too harsh:D ...!!)

Reply:
I think it's fine to write scripts or notebooks that don't use Scipp for a specific purpose like looking at data from an experiment, but they don't need to make it into the Ess reduction software.

The fact that people don't want to use Scipp at jparc is irrelevant I think? I would say that's their loss. Our task is to develop the software for Ess.